CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ If you are updating tests and just want to run a single test to check it, you
 can use this syntax to run it exactly as the test harness would:
 
 ```text
-$ python tools/test.py -v --mode=release parallel/test-stream2-transform
+$ python tools/test.py --mode=release parallel/test-stream2-transform
+```
+
+If you want to check the other option, please refer the help with option
```

Comment:
Nit: take this with a grain of salt as I'm not a native speaker but maybe it better to use "the other options, please refer to the help with the option".